authors/apps/bookmarks/views.py

```diff
@@ -0,0 +1,105 @@
+from rest_framework.generics import CreateAPIView, ListAPIView, RetrieveAPIView, DestroyAPIView
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.exceptions import NotFound
+from authors.apps.articles.models import Article
+from .serializers import BookmarkSerializer
+from .models import Bookmark
+
+
+class GetBookMarks:
+
+    @staticmethod
+    def get_bookmarks(req):
+        try:
+            bookmarklist = Bookmark.objects.all()
+            return bookmarklist
+        except Bookmark.DoesNotExist:
```

Comment:
Return a something meaningful here and we can easily relate with.

Reply:
okay, let me fix this Asap. Thanks for the observation

Reply:
The method is not intended to return any message because I call it in the create method in my bookmark class. It is supposed to return a list of bookmarks if there any and if there are no bookmarks it returns None so that the create method can create a bookmark but if a list of bookmarks is returned, I loop through it and do some checks to ensure that the bookmark the user does not create the same bookmark.